### ISSUE TITLE
Nav menu

### ISSUE THREAD

**Issue by kelvinlockwood** [kelvinlockwood/kelvinlockw.xyz#28]
We currently have a sticky menu but as you scroll it leaves us with this dead space at the top of the page...

<img width="1412" alt="Screenshot 2021-03-24 at 13 39 33" src="https://user-images.githubusercontent.com/11790973/112320156-a7a4fe80-8ca6-11eb-81bd-ed424980b0a2.png">

If we ensure that the nav menu items persist in the gutter space they won't clash with the content of the page, meaning we can display text right to the top of the page.

To achieve this the menu items need to be stacked vertically so that they comfortably fit within the right hand gutter, it will look something like this...

<img width="62" alt="Screenshot 2021-03-24 at 13 33 34" src="https://user-images.githubusercontent.com/11790973/112319010-8a236500-8ca5-11eb-9154-937ef8cecd5e.png">

**Comment by kelvinlockwood** [comment]
**Mobile screens**

We can make the nav menu responsive for mobile screens be following the tutorial here...
https://www.themes.dev/blog/responsive-navigation-menu-tailwind-css/

No need for a 'Home' link. Just Articles, Notes and Work.

<img width="516" alt="Screenshot 2021-03-24 at 15 35 40" src="https://user-images.githubusercontent.com/11790973/112338299-99f77500-8cb6-11eb-83ff-f5f12739c0d4.png">

It might be better to place the tags below these links and remove them from the footer (for mobile only).

**Comment by MoNouri97** [comment]
so we are getting rid of the progress bar in the nav ?

**Comment by kelvinlockwood** [comment]
Yes we don't need the progress bar but the nav bar should still be sticky at the top with 100% opacity so that the items in the nav bar effectively floating above the background.

**Comment by kelvinlockwood** [comment]
The sticky nav bar / vertical menu idea should only work on XL and 2XL browser widths.

For MD and LG it can work as it currently does...

<img width="1108" alt="Screenshot 2021-04-02 at 00 04 02" src="https://user-images.githubusercontent.com/11790973/113362734-f279e280-9346-11eb-9d8f-e805db2251e5.png">
